packages/components/addon/components/hds/table/index.js

```diff
@@ -1,32 +1,50 @@
 import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { guidFor } from '@ember/object/internals';
+import { tracked } from '@glimmer/tracking';
 
-/**
- * @class HdsTable
- */
 export default class HdsTableIndexComponent extends Component {
-  /**
-   * Sets the data source that gets yielded by the `:body` named block.
-   * @argument model
-   * @type {array}
-   */
-  /**
-   * Specifies which field the `@model` is sorted by. This **does not perform a sort**.
-   * It is only used to set states on the contextual `SortBy` components.
-   * @argument sortProperty
-   * @type {string}
-   */
-  /**
-   * Specifies which direction the `@model` is sorted. Descending when `true`, ascending when
-   * `false`. This **does not perform a sort**. It is only used to set states on the
-   * contextual `SortBy` components.
-   * @argument sortDescending
-   * @type {boolean}
-   */
+  a11yMessageId = 'table-a11y-message-' + guidFor(this);
```

Comment:
question: I might be missing something, but is the purpose of this `id` only informational (as I can't see it being referenced somewhere)

thought: we're only using `guidFor` in a few places at the moment and I can see how it can be helpful to put the message in a relation with the table; wondering if appending `hds-` (e.g. `hds-table-a11y-message`) would make it even more obvious who generates/manages this element.